File: parity/corpora/language-matrix/dart/runner.dart
import 'worker.dart';
import 'package:flutter/material.dart';

class Runner extends Service {
  String execute(String value) => process(value);
}

class WorkerPanel extends StatefulWidget {
  const WorkerPanel({super.key});

  @override
  State<WorkerPanel> createState() => _WorkerPanelState();
}

class _WorkerPanelState extends State<WorkerPanel> {
  final Service service = Service();

  @override
  Widget build(BuildContext context) {
    return StreamBuilder<String>(
      stream: const Stream<String>.empty(),
      builder: (context, snapshot) => Text(service.process(snapshot.data ?? '')),
    );
  }
}

File: parity/corpora/language-matrix/php/Runner.php
<?php
namespace MatrixRuntime;

require_once __DIR__ . '/Contracts.php';

class Runner extends Service
{
    public function execute(string $value): string
    {
        return $this->process($value);
    }
}

class Provider
{
    public function register(): void
    {
        $this->app->bind(Worker::class, Service::class);
    }
}
